src/main/java/com/google/codeu/servlets/MapsServlet.java

```diff
@@ -0,0 +1,90 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+
+
+/**
+     * Returns boba locations data as a JSON array, e.g. 
+     * [{"lat": 38.4404675, "lng": -122.7144313}]
+     */
+
+@WebServlet("/boba-locations")
+public class MapsServlet extends HttpServlet {
+
+    JsonArray bobaLocationsArray;
+    @Override
+    public void init() {
+        bobaLocationsArray = new JsonArray();
+        Gson gson = new Gson();
+        Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+            "/WEB-INF/boba-locations.csv"));
+        while(scanner.hasNextLine()) {
+            String line = scanner.nextLine();
+            String[] cells = line.split(",");
+            double lat = Double.parseDouble(cells[0]);
+            double lng = Double.parseDouble(cells[1]);
+
+            bobaLocationsArray.add(gson.toJsonTree(new bobaLocations(lat, lng)));
+        }
+        scanner.close();
+    }
+
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
+    throws IOException {
+        response.setContentType("application/json");
+        response.getOutputStream().println(bobaLocationsArray.toString());
+    }
+
+    private static class bobaLocations {
+        double lat;
+        double lng;
+    
+
+        private bobaLocations(double lat, double lng) {
+        this.lat = lat;
+        this.lng = lng;
+        }
+    }   
+}
+// test above code by runnning on development server 
+    // navigate to http://localhost:8080/boba-locations
+
+    
+
+
+
+
+
+
+
```

Comment:
it is not ideal to have commented out code in your pull requests, but it's fine for now ; make sure you move this code to a servlet or remove it later :)